### DISCUSSION TITLE
Can the "positioning-region" div inside a fluent-radio-group be styled?

### DISCUSSION THREAD

**Discussion by florianwachs** [microsoft/fluentui-blazor#1001] [Q&A] [answered]
This is the default behavior of the radio group with horizontal orientation:
![image](https://github.com/microsoft/fluentui-blazor/assets/2216547/e134623c-f6e7-4d7c-b781-28a3f9b80654)

my container is too small for all items so it breaks.

If I would be able to style the div inside the shadowroot with the class `positioning-region`, I could use the following css:

```css
display:grid;
grid-template-columns: 1fr 1fr;
```

which leads to this:
![image](https://github.com/microsoft/fluentui-blazor/assets/2216547/61b59c21-b2c5-4995-8e49-ec0dbd3d8304)

is there a way to do this?

Thank you very much for your time.

**Comment by vnbaaij** [comment, marked as answer]
In general, you can target anything in the shadow dom with the :part selector. If you search through the repo, you should find some examples

**Reply by florianwachs** [reply]
Thank you very much, I was not aware of the :part selector, I will check it. By the way I really love Fluent Blazor. It is so high quality! Thanks to you and the team. Having a real productivity blast

**Reply by vnbaaij** [reply]
That's great to hear. Thanks and spread the word!
